src/main/java/com/pirimid/cryptotrade/DTO/TradeDto.java

```diff
@@ -0,0 +1,31 @@
+package com.pirimid.cryptotrade.DTO;
+
+import com.pirimid.cryptotrade.model.Side;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class TradeDto {
+    private String tradeId;
+    private String exchangeOrderId;
```

Comment:
is `exchangeOrderId` is orderId we are gettinfg from exchange.

Reply:
yes